### ISSUE TITLE
Welsh Language Accessibility Tweaks

### ISSUE THREAD

**Issue by abbott567** [dwp/design-system#330]
## Related documentation

[Toggle to Welsh pattern](https://design-system.dwp.gov.uk/patterns/toggle-to-welsh)

## Suggestion
The content mentions that you could fail [3.1.2 Language of parts](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts) by getting the lang tag on the links incorrect, which is true, but it doesn't mention that you can fail [3.1.1 Language of page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page) if you don't get the lang attribute correct on the `<html>` element. 

I'd like to suggest this paragraph be tweaked to something along the lines of:
> *To meet WCAG 2.1 success criteria [3.1.1 Language of page](https://www.w3.org/WAI/WCAG21/Understanding/language-of-page) and [3.1.2 Language of parts](https://www.w3.org/WAI/WCAG21/Understanding/language-of-parts), services should make sure the language attribute has been applied properly. This helps things like screen readers to pronounce the content correctly.*

Also, the content kind of implies that the lang attribute will just always be correct on the `<html>` element. For example: `In the English version of the service the page’s default language will be specified as English.` It might not always be specified. the developer should really make sure it is.

So, I'd also like to suggest strengthening the language around the lang attribute on the `<html>` element to something like the following:
> *In the English version of the service, the page’s default language must be set to English with `<html lang="en">` and you must set the correct language on the toggle link for ‘Cymraeg’ with `lang="cy"`.*
> 
> *In the Welsh version of the service, the page’s default language must be set to Welsh with `<html lang="cy">` and you must set the correct language on the toggle link for ‘English’ with `lang="en"`.*

Hope this makes sense!

Thanks,
Craig

**Comment by martinwake** [comment]
@abbott567 Thanks Craig, we've rewritten this section to strengthen the guidance as you suggest.

**Comment by martinwake** [comment]
@abbott567 I was looking at this issue on the GOV.UK backlog: https://github.com/alphagov/govuk-design-system/pull/1632

Should we also be using `hreflang="cy"`?

**Comment by abbott567** [comment]
Hi @martinwake - hreflang is for search engines. So, it will let Google know which language the target page is in so when it indexes them it localises that version for people it identifies as speaking that language.

Probably makes sense to add it. We don't lose anything by adding it, but for some users it might mean they find the right page via Google.
